models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -656,44 +686,37 @@ aeif_cond_beta_multisynapse_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = y[ S::V_M ];
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
   const double& w = y[ S::W ];
 
   double I_syn = 0.0;
 
-  for ( size_t i = 0; i < ( node.P_.num_of_receptors_
-                            * S::NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR );
-        i += S::NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR )
+  for ( size_t i = 0; i < node.P_.num_of_receptors; ++i )
   {
-    double g = y[ S::G + i ];
-    if ( g > 0 )
-    {
-      I_syn += g * ( node.P_.E_ex - V ); // g>0, E_ex>V => I_syn increases
-    }
-    else
-    {
-      I_syn += g * ( V - node.P_.E_in ); // g<0, V>E_in => I_syn decreases
-    }
+    size_t j = i * S::NUMBER_OF_STATES_ELEMENTS_PER_RECEPTOR;
+    // I_syn = I_syn + g*(E_rev - V)
```

Comment:
Maybe move this comment above the `I_syn = 0` initialization and make the summation explicit: `I_syn = - sum_k g_k (V - E_rev_k)`.